docs/pages/access-graph/getting-started-on-prem.mdx

```diff
@@ -0,0 +1,122 @@
+---
+title: Getting started with Teleport Access Graph (TAG)
+---
+
+<Notice type="note">
+Teleport Access Graph is provided by default to users of the Teleport Team plan.
+Visit the [signup page](https://goteleport.com/signup/) to begin your free&nbsp;trial.
+</Notice>
+
+## Prerequisites
+
+<Tabs>
+<TabItem
+  scope={["enterprise"]} label="Self-Hosted Enterprise">
+
+- A running Teleport Enterprise cluster v15.0.0 or later.
+- Docker version v(=docker.version=) or later.
```

Comment:
Technically, docker is not required. Any container runtime will work.

Reply:
That's true. However, our other guides that use containerized images seem to point towards Docker in a similar way, e.g. https://goteleport.com/docs/management/export-audit-events/fluentd/#prerequisites

Not sure how we'd change up instructions to `docker run`. Maybe somehow say "Docker or other container runtime, but we provide instructions for Docker here" 🙂

Reply:
Let's leave Docker for now. I'd assume that most people understand how to run a container.